curiouslearning/FeedTheMonsterJS
# Feat: only appear on next playable level (FM-271)

# Changes
- Making sure that pulse effect only activated on next playable level and not on previous levels that already played.

# How to test
- Open the start screen of FTM.
- Tap or click any part of the start screen to proceed to the next screen.
- Play a previous level, then exit to return to the level selection screen.
- Ensure the pulse animation is on the next playable level.
- When hitting refresh or reopening the app, confirm that it navigates to the page where the next playable level is located.

Ref: [FM-271](https://curiouslearning.atlassian.net/browse/FM-271)


[FM-271]: https://curiouslearning.atlassian.net/browse/FM-271?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
